dl/src/main/scala/com/intel/analytics/bigdl/optim/SGD.scala

```diff
@@ -101,40 +102,64 @@ class SGD[@specialized(Float, Double) T: ClassTag](implicit ev: TensorNumeric[T]
 }
 
 object SGD {
-  trait LearningRateSchedule {
+
+  /**
+   * A trait for user defined hyper parameter scheduler.
+   * Hyper parameters should be defined in config Table, supported parameters are:
+   * learningRate, default is 1e-3
+   * weightDecay, default is 0.0
+   * momentum, default is 0.0
+   * dampening, default is momentum
+   * nesterov, default is false
+   * learningRates, default is null. If defined, size must be the same as gradient Tensor.
+   * weightDecays, default is null. If defined, size must be the same as gradient Tensor.
+   *
+   * Please don't overwrite the base hyper parameters list above, and use below mapping
+   * to define the current hyper parameters.
+   *
+   * config("clr") = current learning rate
+   * config("wd") = current weight decay
+   * config("mom") = current momentum
+   * config("damp") = current dampening
+   * config("cntv") = current nesterov
+   * config("lrs") = current learning rates
+   * config("wds") = current weight decays
+   *
+   * Notice: the clr, wd, mom and damp should be double value.
+   */
+  trait HyperParameterScheduler {
     def updateHyperParameter(config : Table, state : Table) : Unit
   }
 
-  case class EpochSchedule(regimes : Array[Regime]) extends LearningRateSchedule {
+  case class EpochSchedule(regimes : Array[Regime]) extends HyperParameterScheduler {
     override def updateHyperParameter(config: Table, state: Table): Unit = {
       val epoch = config[Int]("epoch")
       for (r <- regimes) {
         if (epoch >= r.startEpoch && epoch <= r.endEpoch) {
           config.add(r.config)
         }
       }
-      config("clr") = -config.get[Double]("learningRate").getOrElse(1e-3)
+      config("clr") = config.get[Double]("learningRate").getOrElse(1e-3)
```

Comment:
why change this? If you want to display a positive learningRate, you should negative it at the display part.